src/main/java/xiaobai/AddEventCommand.java

```diff
@@ -0,0 +1,35 @@
+package xiaobai;
+
+public class AddEventCommand extends Command {
+    private final String desc, start, end;
+
+    /**
+     * Creates an AddEventCommand with the given description, start, and end time.
+     *
+     * @param desc Description of task.
+     * @param start Start time of event.
+     * @param end End time of event.
+     */
+    public AddEventCommand(String desc, String start, String end) {
+        this.desc = desc; this.start = start; this.end = end;
+    }
+
+    /**
+     * Creates an Event task.
+     * Saves the updated task list.
+     *
+     * @param tasks Task list.
+     * @param ui User interface.
+     * @param storage Storage handler.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws XiaoBaiException {
+        if (desc == null || desc.isBlank() || start == null || start.isBlank() || end == null || end.isBlank()) {
```

Comment:
While the line is within the limit, would it be better to break the lines into separate cases to make it easier to see?
```suggestion
        if (desc == null || desc.isBlank() 
                || start == null || start.isBlank() 
                || end == null || end.isBlank()) {
```